layouts/partials/search-bar.html

```diff
@@ -1,8 +1,7 @@
 <div class="navbar-item is-hidden-touch">
   <p class="control has-icons-left">
-    <input class="input is-rounded" type="text" placeholder="Search" id="search-input">
-    <span class="icon is-small is-left">
-      <ion-icon name="search-outline"></ion-icon>
-    </span>
+    <div class="container">
+      <div id="docsearch"></div>
```

Comment:
I guess this change is from the their example? But will this impact CSS styling change?

Reply:
Yeah, It's from Docserach official website.
I am not good at web UI and need someone's double check, but it seams the deploy-preview looks the same as original one except the style of search bar.

Reply:
You can run `make serve` to see locally, but I believe the style will be impacted. Or, we can check it together tomorrow.

Reply:
After discussing, the style will be controlled by docsearch styling instead, so we don't need to worry about it.

Reply:
Yeah, I did use `make serve` to check the changes.
Modified. Thanks. only `<div id="docsearch"></div>`